### PR TITLE
Cosmetic styles for login page and props errors were removed

### PR DESCRIPTION
**Pull request by serhiirubets**

### CODE REVIEW

File: src/components/authorization/login/login-form.js [hunk 1]
```diff
@@ -22,6 +24,10 @@ function mapDispatchToProps(dispatch) {
   };
 }
 
+function checkCorrectInputData(email, password) {
```
Comment (httpEugene) [inline src/components/authorization/login/login-form.js]: I would call this `isEmailCorrect`

File: src/components/field/field.js [hunk 2]
```diff
@@ -7,18 +7,28 @@ import styles from './styles';
 export default class Field extends PureComponent {
   static propTypes = {
     placeholder: PropTypes.string,
-    type: PropTypes.strategy,
+    type: PropTypes.string,
     handleForm: PropTypes.func,
+    name: PropTypes.string,
+  };
+
+  onChangeHandler = (text, name, cb) => {
```
Comment (httpEugene) [inline src/components/field/field.js]: `onInputChange`

File: src/components/authorization/login/login-form.js [hunk 3]
```diff
@@ -41,14 +41,52 @@ export default class LoginForm extends PureComponent {
...
+    wrongEmailValue = this.state.username !== 'test@gmail.com';
   };
 
-  handleFormSubmit = () => {
-    if (checkCorrectInputData(this.state.username, this.state.password)) {
+  checkPassword = () => {
+    wrongPassword = this.state.password !== 'test';
+  };
+
+  isCredentialsCorrect = () => {
```
Comment (httpEugene) [inline src/components/authorization/login/login-form.js]: ` checkEmailValue = () => {
   return this.state.username !== 'test@gmail.com';
    };

 checkPassword = () => {
   return this.state.password !== 'test';
 };
 
 isCredentialsCorrect = () => {
   this.setState({
     errors: {
       wrongEmailFormat: !isEmail(this.state.username),
       wrongEmailValue: checkEmailValue(),
       wrongPassword: checkPassword(),
     },
   });
 };`

File: src/components/drawer/styles.js [hunk 4]
```diff
@@ -18,7 +18,6 @@ const styles = StyleSheet.create({
     borderTopWidth: 0,
     borderBottomWidth: 1,
     borderBottomColor: '#fff',
-    color: '#000',
```
Comment (l0lka) [inline src/components/drawer/styles.js]: Use variables please

File: src/components/field/styles.js [hunk 5]
```diff
@@ -1,9 +1,12 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
-  field: {
-    color: '#000',
-    fontSize: 24,
+  input: {
+    fontSize: 14,
+    color: '#666',
```
Comment (l0lka) [inline src/components/field/styles.js]: Use variables please

**Pull request merged**